galley/pkg/server/settings/args.go

```diff
@@ -143,6 +143,12 @@ type Args struct { // nolint:maligned
 	// keep-alive options for the MCP gRPC Server.
 	KeepAlive *keepalive.Options
 
+	// Number of seconds to be used for rate limiting in the source server
+	SourceServerRatelimiter uint
+
+	// Number of seconds to be used for rate limiting in the MCP sources
```

Comment:
isn't this burst size not seconds?

Reply:
yeah, you are right. I got it mixed up by the frequency (rate.Every). I will update.